hat/assets/js/apps/Iaso/domains/completenessStats/hooks/useCompletenessStatsColumns.tsx

```diff
@@ -77,10 +81,49 @@ export const useCompletenessStatsColumns = (params: any) => {
                 accessor: 'forms_filled_direct',
                 sortable: false,
                 Cell: settings => (
-                    <span>
-                        {settings.row.original.forms_filled_direct ?? '--'}/
-                        {settings.row.original.forms_to_fill_direct ?? '--'}
-                    </span>
+                    <Box>
+                        <div
```

Comment:
You can use Box instead, and use `display`, `alignItems` props from `Box`, not sure de `verticalAlign` is usefull here...

Reply:
I can use the Box, but if I use the props i.o `style`, the layout breaks